les/peer.go

```diff
@@ -58,7 +58,7 @@ type peer struct {
 	version int    // Protocol version negotiated
 	network uint64 // Network ID being on
 
-	announceType, requestAnnounceType uint64
+	requestAnnounceType uint64
```

Comment:
It is ok to unify these two fields since you either announce to or request announcements from a peer but then I'd call it announceType.